src/api-reference/budget/budget-category.md

```diff
@@ -3,184 +3,554 @@ title: Budget Category
+  * [Budget Category](#budgetCategory)
+  * [Expense Type](#expenseType)
+  * [Error Response](#errorResponse)
+  * [Error Message](#errorMessage)
 
-- [Getting Started](#overview)
-- [Fiscal Year](#overview)
-- [Budget Item](#overview)
-- [Budget Tracking Fields](#overview)
-- [Budget Adjustments](#overview)
+## Version
 
-# Budget Category - PRE-RELEASE
+4.0
 
-- [Overview](#overview)
-- [Retrieve all Budget Categories](#getall)
-- [Retrieve a Budget Category](#get)
-- [Create/Update a Budget Category](#post)
-- [Remove a Budget Category](#delete)
-- [Retrieve all valid Expense Types](#getExpTypes)
-- [Schema](#schema)
+## <a name="getall"></a>GET all Budget Categories
 
-## Overview
+Retrieve a list of all budget categories
 
-The new Budget Service API is in **PRE-RELEASE**. If you are interested in using the Budget Service API, then please contact your account manager for further details.
+### Scopes
 
-This resource is used to retrieve and update budget categories which are collections of expense types used for budget
-matching. Each budget item header may have one budget category. If it does, only line items with expense types
-contained in that budget category will be accumulated to the budget.
+Name | Description
+---|---
+`budgetitem.write`|Create/update/delete access to budget data
+`budgetitem.read`|Read access to budget data
 
-## Version
+### Request
 
-4.0
+#### Headers
 
-## <a name="getall"></a>Retrieve all Budget Categories
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
 
-    GET  /budget/v4/budgetCategory
+#### Parameters
 
-HTTPie:
+N/A
 
-```shell
-http https://us.api.concursolutions.com/budget/v4/budgetCategory 'Authorization:Bearer {YOUR ACCESS TOKEN}'
+##### URI Template
+
+```http
+GET /budget/v4/budgetCategory
 ```
 
-### Parameters
+### Response
 
-    N/A
+#### Status Codes
 
-### Response
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1) Successful call, response is in body.
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1) The request was determined to be invalid by the server.
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3) The user does not have the necessary permissions to perform the request.
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1) Error message in response body
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5) Error message in response body
+
+#### Headers
+
+* `concur-correlationid` (Optional) is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+
+#### Payload
+
+Array of [Budget Category](#budgetCategory)
+
+### Example
+
+#### Request
+```http
+GET https://us.api.concursolutions.com/budget/v4/budgetCategory
+Authorization: Bearer: {YOUR ACCESS TOKEN}
```

Comment:
Should be: `Authorization: Bearer: {token}`

Reply:
True.  Also, it shouldn't have a colon.  (i.e. "Bearer {token}" not "Bearer: {token}")